site/tests/universal/blockpage.test.ts

```diff
@@ -0,0 +1,58 @@
+import { expect, test } from "playwright-test-coverage";
+
+const codeBlockUrl = "/@hash/blocks/code";
```

Comment:
We hardcode this because `blocks-data.json` is not present in the vercel deployment. There's an assertion in the test to confirm if this is correct, so in the event the block url changes, the test will fail and we can always update